src/Bookings.js

```diff
@@ -1,21 +1,69 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import NewBookingForm from "./NewBookingForm.js";
+import FakeBookings from "./data/fakeBookings.json";
 
-const Bookings = () => {
-  const search = searchVal => {
-    console.info("TO DO!", searchVal);
+function Bookings() {
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [bookings, setBookings] = useState([]);
+  const [typedSearchValues, setTypedSearchValues] = useState([]);
+  
+  const search = (searchVal) => {
+    setTypedSearchValues(searchVal);
   };
 
+  function addNewBooking(newBooking){
+    setBookings([...bookings, newBooking])
+  }
+
+  const filteredBookings = bookings.filter((customerName) => {
```

Comment:
I don't think we have named this `customerName` variable very well 🤔 
What is this value actually? It's a `booking` from our `bookings` array! So we should call it `booking` 😎

Reply:
I understand what you mean. I will update it once I can fetch the API again so I can test that it is working.... But I get your point here.